packages/fxa-content-server/app/scripts/views/password_strength/password_strength_balloon.js

```diff
@@ -18,10 +18,6 @@ import OneVisibleOfTypeMixin from '../mixins/one-visible-of-type-mixin';
 import Template from '../../templates/partial/password-strength-balloon.mustache';
 import XSS from '../../lib/xss';
 
-// Allow the balloon to stay visible for a bit so that
-// the user can see all the criteria were met.
-const DELAY_BEFORE_HIDE_MS = 750;
```

Comment:
We no longer need this and related code since the balloon is now hidden when the user changes focus instead of waiting 750ms.